climada/util/value_representation.py

```diff
@@ -29,17 +29,21 @@
 
 LOGGER = logging.getLogger(__name__)
 
-ABBREV = {1:'',
-          1000: 'K',
-          1000000: 'M',
-          1000000000: 'Bn',
-          1000000000000: 'Tn'}
+ABBREV = {
+    1:'',
+    1000: 'K',
+    1000000: 'M',
+    1000000000: 'Bn',
+    1000000000000: 'Tn'
+    }
 
 
 def sig_dig(x, n_sig_dig = 16):
     """
     Rounds x to n_sig_dig number of significant digits.
-    Examples: 1.234567 -> 1.2346, 123456.89 -> 123460.0
+    0, inf, Nan are returned unchanged.
+    Examples: n_sig_dig = 5
+        1.234567 -> 1.2346, 123456.89 -> 123460.0
```

Comment:
I cannot find my previous comment now to this, so maybe I cross-post here:
why does the sig_dig in this example correctly round up the ...56.89 to ...60.0, but wrongly rounds down 1.234567 to 1.2346?

Reply:
This is the expected behaviour. This returns a certain number of "significant digits", it does not round. 

For ` 1.234567`, the first 5 digits are ` 1.2345`. Then, the last one is rounded up because the 6th digit is `6>5`. Al the remaining digits are set to 0.

Reply:
ah ah, my bad. i mis-read it.

Reply:
> What do you mean?
I don't understand what you don't understand, I'm afraid.

Reply:
```
impact_metrics = self.samples_df.apply(_map_impact_calc) ?
```
I still think the mapping cannot be done in this syntax, be I will give it a shot since it would more elegant.

Reply:
OK, I think this does not work. For the implementation, I need to access the `self.calc_eai_exp` and `self.calc_at_event`. However, if I use `self.samples_df.apply(_map_impact_calc)`, the `self` si not passed. I tried it a bit, but did not succeed. I do not really see a good reason to change that either. So, unless you insists (which is your right), I would leave it as is.